standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -9840,9 +9845,9 @@ public FireEventResponse fire_listener_event(FireEventRequest rqst) throws TExce
                 .get(MetaStoreEventListenerConstants.DB_NOTIFICATION_EVENT_ID_KEY_NAME)));
       } else {
         String msg = "Reload event id not generated for ";
-        if (event.getPartitionObj() != null) {
-          msg += "partition " + Arrays
-                  .toString(event.getPartitionObj().getValues().toArray()) + " of ";
+        if (event.getPartitions() != null) {
```

Comment:
if the partitions are large, will the message pollute the whole log file? if we really need it, how about only log the part_vals?